app/models/concerns/curation_concerns/suppressible.rb

```diff
@@ -1,20 +1,25 @@
 module CurationConcerns
   # Workflow considerations
-  module Publishable
+  module Suppressible
     extend ActiveSupport::Concern
 
     included do
       # This holds the workflow state
       property :state, predicate: Vocab::FedoraResourceStatus.objState, multiple: false
+    end
+
+    def deactivate
+      self.state = Vocab::FedoraResourceStatus.inactive
+    end
 
-      class_attribute :state_workflow, instance_writer: false
-      self.state_workflow = StateWorkflow
+    def activate
+      self.state = Vocab::FedoraResourceStatus.active
```

Comment:
I think this should remain in the ActivateObject service class. Why are you moving this here. The responsibity of the model is persisting data. The responsibility of the service object is knowing how to activate. Each object should have a single responsibility.